seismicpro/const.py

```diff
@@ -19,6 +19,7 @@
 # Custom headers optionally added by SeismicPro to those from SEG-Y file
 HDR_DEAD_TRACE = 'DeadTrace'
 HDR_FIRST_BREAK = 'FirstBreak'
+HDR_TRACES_POS = 'TRACES_POS'
```

Comment:
```suggestion
HDR_TRACE_POS = 'TRACE_POS'
```
I'd at least remove "s" to match other headers like "FirstBreak" or "offset" which refer to a single trace.
Probably I'd also rename it to `TRACE_POS_IN_SURVEY`. It's used only in a couple of places, so a longer but more explicit name seems acceptable.